packages/pluggableWidgets/image-viewer-web/CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Changelog
+All notable changes to this widget will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+### Changed
+- Refactored into pluggable widget.
```

Comment:
I would say: "Converted to a pluggable widget." as "refactoring" isn't relevant for customers.

Reply:
Changed 👍